package.json

```
@@ -8,6 +8,7 @@
     "query-string": "7.0.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
+    "react-paginate": "^8.1.2",
```

Please freeze package

```suggestion
    "react-paginate": "8.1.2",
```